benchmarks/templates/benchmarks/invalid_zip.html

```diff
@@ -0,0 +1,18 @@
+{% extends 'benchmarks/base.html' %}
+
+{% block content %}
+<section id="success" class="container center">
+  <h2>Uh Oh! Your submission package was not in the correct format.</h2>
```

Comment:
```suggestion
  <h2>Your submission package was not in the correct format.</h2>
```
Let's keep some professionalism